legislators-current.yaml

```diff
@@ -39594,7 +39594,7 @@
     class: 2
     state_rank: junior
     party: Republican
-    url: http://www.rounds.senate.gov
+    url: https://www.rounds.senate.gov
```

Comment:
The last commit was an attempt to show what a sample change would look like with this approach.  I edited the top PR comment to indicate that this branch shouldn't be merged as is too, i can make a new branch / pr when we agree on an approach.